Form-Controls/index.html

```diff
@@ -17,11 +17,51 @@ <h1>Product Pick</h1>
         <!-- 
           try writing out the requirements first as comments
         this will also help you fill in your PR message later-->
+        <form>
+          <p>
+            <label for="name">Name:</label>
+            <input type="text" id="name" name="user_name" />
+          </p>
+          <p>
+            <label for="mail">Email:</label>
+            <input type="email" id="mail" name="user_email" />
+          </p>
+
+           <!-- T-Shirt Color -->
+        <label for="color">Colour</label>
+        <select id="color" name="color" required>
+            <option value="red">Red</option>
+            <option value="blue">Blue</option>
+            <option value="green">Green</option>
+        </select>
+        <br><br>
+
+        <!-- T-Shirt Size -->
+         <div>
+            <label for="size">Size</label>
+            <select id="size" name="size" required>
+            <option value="XS">XS</option>
+            <option value="S">S</option>
+            <option value="M">M</option>
+            <option value="L">L</option>
+            <option value="XL">XL</option>
+            <option value="XXL">XXL</option>
+            </select>
+           </div>
+
+          <h1>
+            <button>Order</button>
+          </h1>
+        </form>
+
       </form>
+    
     </main>
     <footer>
       <!-- change to your name-->
-      <h2>By HOMEWORK SOLUTION</h2>
+      <h2>By Maryna Romanova</h2>
```

Comment:
I don't think that heading level 2 is appropriate in this context. Please, update.